spec/features/translations/poll_question_answers_spec.rb

```diff
@@ -0,0 +1,138 @@
+
+          select('Español', from: 'locale-switcher')
+          expect(page).to have_content "Respuesta en Español"
+          expect(page).to have_content "Descripción en Español"
+
+          select('Français', from: 'locale-switcher')
+          expect(page).to have_content "Answer en Français"
+          expect(page).to have_content "Description en Français"
+        end
+
+        scenario "Update a translation", :js do
+          visit @edit_answer_url
+
+          click_link "Español"
+          fill_in 'poll_question_answer_title_es', with: 'Pregunta correcta en Español'
+          fill_in_ckeditor 'poll_question_answer_description_es', with: 'Descripción correcta en Español'
+
+          click_button 'Save'
+          expect(page).to have_content "Changes saved"
+
+          expect(page).to have_content("Answer in English")
+          expect(page).to have_content("Description in English")
+
+          select('Español', from: 'locale-switcher')
+          expect(page).to have_content("Pregunta correcta en Español")
+          expect(page).to have_content("Descripción correcta en Español")
+        end
+
+        scenario "Remove a translation", :js do
+          visit @edit_answer_url
+
+          click_link "Español"
+          click_link "Remove language"
+
+          expect(page).not_to have_link "Español"
+
+          click_button "Save"
+          visit @edit_answer_url
+          expect(page).not_to have_link "Español"
+        end
+
+        context "Globalize javascript interface" do
+
+          scenario "Highlight current locale", :js do
+            visit @edit_answer_url
+
+            expect(find("a.js-globalize-locale-link.is-active")).to have_content "English"
+
+            select('Español', from: 'locale-switcher')
+
+            expect(find("a.js-globalize-locale-link.is-active")).to have_content "Español"
+          end
+
+          scenario "Highlight selected locale", :js do
+            visit @edit_answer_url
+
+            expect(find("a.js-globalize-locale-link.is-active")).to have_content "English"
+
+            click_link "Español"
+
+            expect(find("a.js-globalize-locale-link.is-active")).to have_content "Español"
+          end
+
+          scenario "Show selected locale form", :js do
+            visit @edit_answer_url
+
+            expect(page).to have_field('poll_question_answer_title_en', with: 'Answer in English')
+
+            click_link "Español"
+
+            expect(page).to have_field('poll_question_answer_title_es', with: 'Respuesta en Español')
+          end
+
+          scenario "Select a locale and add it to the poll form", :js do
+            visit @edit_answer_url
+
+            select "Français", from: "translation_locale"
+
+            expect(page).to have_link "Français"
+
+            click_link "Français"
+
+            expect(page).to have_field('poll_question_answer_title_fr')
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
Since you started working on this, we refactored all of the common admin translation tests out to RSpec shared examples: https://github.com/consul/consul/pull/2903. So you can actually replace most of these tests with `it_behaves_like "translatable", ...`  (see #2903). Sorry you had to go through the trouble of writing them! :/

Reply:
Done https://github.com/consul/consul/pull/2914/commits/df9955e0c60e766abf8a0046f96dbce3501b7bba

Reply:
I left the original specs for the answers cause using the shared code was giving me random errors 
```
Failure/Error: fill_in field_for(field, :fr), with: text_for(field, :fr)
     
     Capybara::ElementNotFound:
       Unable to find visible field "poll_question_answer_description_fr" that is not disabled
```
while the original specs doesn't